CLDSRV-343: use bucket name for backend tagging operations

williamlardier opened the pull request:
Fixes [ARTESCA-6254](https://scality.atlassian.net/browse/ARTESCA-6254)

A function for multipple backends tagging APIs was passing a bucket object instead of its name, causing a wrong parameter to be sent to the backend (Ring/AWS). This led to failures when trying to tag objects in these locations.

No test added here, but re-enabled some CEPH tests that were skipped, and related to the use of these APIs for multiple backends.

[ARTESCA-6254]: https://scality.atlassian.net/browse/ARTESCA-6254?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Seragonia committed:
CLDSRV-343: use bucket name for backend tagging operations

bert-e commented:
# Hello williamlardier,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://bert-e.scality.net/scality/cloudserver/doc/user).





*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue CLDSRV-343 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `8.7.8`


Please check the `Fix Version/s` of CLDSRV-343, or the target
branch of this pull request.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers

Seragonia committed:
CLDSRV-343: enable back some CEPH backend tests

These tests also cover the ObjectTagging API with multiple backend.
Enabling them back will allow us to better avoid issues like this
in the future.

Seragonia committed:
CLDSRV-343: bump cloudserver to the next version

ghost commented:
/approve

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/8.7`



The following branches will **NOT** be impacted:

* `development/7.10`
* `development/7.4`
* `development/7.70`
* `development/8.4`
* `development/8.5`
* `development/8.6`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/8.7`



The following branches have **NOT** changed:

* `development/7.10`
* `development/7.4`
* `development/7.70`
* `development/8.4`
* `development/8.5`
* `development/8.6`



Please check the status of the associated issue CLDSRV-343.

Goodbye williamlardier.
